gapic-generator/test/test_helper.rb

```diff
@@ -15,7 +15,9 @@
 # limitations under the License.
 
 $LOAD_PATH.unshift File.expand_path("../lib", __dir__)
+$LOAD_PATH.unshift File.expand_path("../test", __dir__)
```

Comment:
This also seems unnecessary. Do we need to `require` any files in the test directory other than this helper?

Reply:
both this and the similar one in cloud are for the purpose of sharing the test resources for the parameter parsing:

`require "gapic/schema/api_test_resources"`

Reply:
moved the shared test resources to shared instead